File: src/main/java/upbrella/be/store/dto/request/SingleBusinessHourRequest.java
package upbrella.be.store.dto.request;

import com.fasterxml.jackson.annotation.JsonFormat;
import lombok.*;

import java.time.DayOfWeek;
import java.time.LocalTime;

@Getter
@Builder
@NoArgsConstructor(access = AccessLevel.PROTECTED)
@AllArgsConstructor
public class SingleBusinessHourRequest {

    private DayOfWeek date;
    @JsonFormat(pattern = "HH:mm")
    private LocalTime openAt;
    @JsonFormat(pattern = "HH:mm")
    private LocalTime closeAt;
}

File: src/main/kotlin/upbrella/be/store/dto/request/SingleBusinessHourRequest.kt
package upbrella.be.store.dto.request

import com.fasterxml.jackson.annotation.JsonFormat
import java.time.DayOfWeek
import java.time.LocalTime

data class SingleBusinessHourRequest(
    val date: DayOfWeek,

    @field:JsonFormat(pattern = "HH:mm")
    val openAt: LocalTime,

    @field:JsonFormat(pattern = "HH:mm")
    val closeAt: LocalTime
)
